NEWS.md

```diff
@@ -1,3 +1,6 @@
+## 1.12.1 2021-12-21
+* Upgrade to RMB 33.0.4 and Log4j 2.16.0 (MODCAL-99)
```

Comment:
The correct Jira is https://issues.folio.org/browse/MODCAL-100 for Juniper (1.12.0 -> 1.12.1).
https://issues.folio.org/browse/MODCAL-99 is for Kiwi (1.13.0 -> 1.13.1): https://github.com/folio-org/mod-calendar/pull/134/files

Reply:
done